app/javascript/dashboard/components/widgets/conversation/FilterItem.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <select v-model="activeValue" class="status--filter" @change="onTabChange()">
+    <option v-for="(value, status) in items" :key="status" :value="status">
+      {{ value['TEXT'] }}
+    </option>
+  </select>
+</template>
+
+<script>
+import wootConstants from '../../../constants';
+import eventListenerMixins from 'shared/mixins/eventListenerMixins';
+import { hasPressedAltAndBKey } from 'shared/helpers/KeyboardHelpers';
+
+export default {
+  mixins: [eventListenerMixins],
+  props: {
+    selectedValue: {
+      type: String,
+      required: true,
+    },
+    items: {
+      type: Object,
+      required: true,
+    },
+    type: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      activeValue: this.selectedValue,
+    };
+  },
+  methods: {
+    handleKeyEvents(e) {
+      if (hasPressedAltAndBKey(e) && this.type === 'status') {
```

Comment:
@pranavrajs @scmmishra @nithindavid Should we handle the keyboard actions here since it has been changed to popover? The keyboard actions should work only if the popover is open. We could move these keyboard actions to the root component, but we need to show the changed filters (status & sort) for the user. WDYT?

https://user-images.githubusercontent.com/12408980/230555063-d6289c2b-b21e-48bd-96ed-371b5ac7c96f.mov

Reply:
We can remove this.

Reply:
Sure.